AirCasting/Dashboard/AirSectionPickerView.swift

```diff
@@ -2,6 +2,7 @@
 //
 
 import SwiftUI
+import Combine
```

Comment:
Why did you add Combine? You don't need it :)

Reply:
Yeah, I've added it when I had the other conception 😅